jenkins/docker/windows/javascript.Dockerfile

```diff
@@ -12,9 +12,7 @@ RUN Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser; `
 	$command='c:\scoop.ps1 -RunAsAdmin'; `
 	iex $command; `
 	del c:\scoop.ps1; `
-	scoop install git shellcheck nodejs-lts cygwin rustup; `
-	scoop bucket add versions; `
-	scoop install versions/python311; `
+	scoop install git shellcheck nodejs-lts cygwin rustup python3; `
```

Comment:
what python version is that? 3.11?

Reply:
Scoop will always install the latest Python version on Windows.  Currently, this is Python 3.12.1.

This is an excellent way to find out when our Python SDK is broken with the latest release 😅